test/unit/spec/MfaVerify_spec.js

```diff
@@ -162,6 +164,8 @@ function (Q,
     var setupOktaPush = _.partial(setup, resAllFactors, { factorType: 'push', provider: 'OKTA' });
     var setupOktaTOTP = _.partial(setup, resVerifyTOTPOnly, { factorType: 'token:software:totp' });
     var setupWebauthn = _.partial(setup, resAllFactors, {  factorType: 'webauthn', provider: 'FIDO' });
+    var setupWebauthnOnly = _.partial(setup, [resRequiredWebauthn, resChallengeWebauthn, resSuccess],
```

Comment:
I think this should be ```_.partial(setup, resRequiredWebauthn, {  factorType: 'webauthn', provider: 'FIDO' });
If you look at line 126 we set there that the next response will be ```resChallengeWebauthn```, so that is taken care of.